frontend/src/app/aethel/aethel.component.ts

```diff
@@ -15,8 +15,10 @@ import {
     styleUrl: "./aethel.component.scss",
 })
 export class AethelComponent implements OnInit {
-    aethelInput = new FormControl<string>("", {
-        validators: [Validators.minLength(3)],
+    form = new FormGroup({
```

Comment:
`FormGroup` basically represents a form. It groups individual `FormControl`s together and offers methods that let you check the validity of an entire form, mark all controls as touched/dirty and dynamically add/remove controls at runtime, to name a few. We do not need these conveniences in our 'form' (which only consists of one control), but it is apparently needed for Angular's `ngSubmit()` to work (and to prevent a page reload on form submission).